sw/device/tests/BUILD

```diff
@@ -1806,6 +1806,28 @@ opentitan_functest(
     ],
 )
 
+opentitan_functest(
+    name = "i2c_host_compass_test",
+    srcs = ["i2c_host_compass_test.c"],
+    cw310 = cw310_params(
+        tags = ["skip_in_ci"],  # Requires the Bob in CI.
```

Comment:
Does this Bob need to be BoB?

Reply:
Ah yes, thanks

Reply:
It should also have been `"manual"`. Fixed.